rocAL/rocAL/include/augmentations/geometry_augmentations/node_resize.h

```diff
@@ -22,19 +22,27 @@ THE SOFTWARE.
 
 #pragma once
 #include "node.h"
+#include "rocal_api_types.h"
 
-class ResizeNode : public Node
-{
+class ResizeNode : public Node {
 public:
     ResizeNode(const std::vector<Image *> &inputs, const std::vector<Image *> &outputs);
     ResizeNode() = delete;
     unsigned int get_dst_width() { return _outputs[0]->info().width(); }
     unsigned int get_dst_height() { return _outputs[0]->info().height_single(); }
     vx_array get_src_width() { return _src_roi_width; }
     vx_array get_src_height() { return _src_roi_height; }
+    void init(unsigned dest_width, unsigned dest_height, RocalResizeScalingMode scaling_mode,
+              std::vector<unsigned> max_size, RocalResizeInterpolationType interpolation_type);
+    void adjust_out_roi_size();
 protected:
     void create_node() override;
     void update_node() override;
 private:
-    vx_array  _dst_roi_width , _dst_roi_height ;
+    vx_array  _dst_roi_width , _dst_roi_height;
```

Comment:
we need to get rid of roi arrays and use single tensor for ROIs when we have tensor implementation